com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -27,6 +27,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed regression from 8.x.x that increased launch times on Android with GLES3. [case 1269119](https://issuetracker.unity3d.com/issues/android-launch-times-increased-x4-from-urp-8-dot-1-0-to-urp-10-dot-0-0-preview-dot-26)
 - Fixed an issue with a render texture failing assertion when chosing an invalid format. [case 1222676](https://issuetracker.unity3d.com/issues/the-error-occurs-when-a-render-texture-which-has-a-certain-color-format-is-applied-to-the-cameras-output-target)
 
+### Added
+- Added real-time Point Light Shadows.
```

Comment:
`### Added` needs to be at the top of the 10.1 list. And there should be only one, there are three right now for some reason. You probably need to merge the latest from master.

Reply:
Thanks, I did that in 0a41462, and later updated versions in e716c26 after a conversation with @oleks-k